process/smartContract/builtInFunctions/changeOwnerAddress.go

```diff
@@ -21,6 +21,11 @@ func NewChangeOwnerAddressFunc(gasCost uint64) *changeOwnerAddress {
 	return &changeOwnerAddress{gasCost: gasCost}
 }
 
+// SetNewGasConfig is called whenever gas cost is changed
+func (c *changeOwnerAddress) SetNewGasConfig(gasCost *process.GasCost) {
+	c.gasCost = gasCost.BuiltInCost.ChangeOwnerAddress
```

Comment:
We need to protect `e.gasCost` with a mutex

Reply:
no need. there is no possibility to call the from different go routines

Reply:
protected all the way